build/webcharts.js

```diff
@@ -1951,6 +1957,25 @@ function drawArea(area_drawer, area_data, datum_accessor, class_match, bind_acce
   return area_grps;
 }
 
+function destroy() {
+  var destroyControls = arguments.length <= 0 || arguments[0] === undefined ? true : arguments[0];
+
+  //run onDestroy callback
+  this.events.onDestroy.call(this);
```

Comment:
correct. it will be called with `chart.destroy()` so `this` is the chart object. See updates in `src/chart/index.js` for implementation details on why this works.